### PR TITLE
AP_Bootloader: Reserve 100 IDs for SYPAQ Systems

### PR DESCRIPTION
**Pull request by sypaq-nexton**
And adjust the reservations for Carbonix and SpektreWorks to reserve 100 IDs instead of 101.

### CODE REVIEW

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -249,6 +249,7 @@ AP_HW_PIXRACER_PERIPH                1402
 AP_HW_SWBOOMBOARD_PERIPH             1403
 
 # IDs 5000-5100 reserved for Carbonix
+# IDs 5101-5199 reserved for SYPAQ Systems
 # IDs 6000-6100 reserved for SpektreWorks
```
[the remark this answers is not in the document]
Comment (Lokesh-Carbonix) [reply Tools/AP_Bootloader/board_types.txt:253]: Yes, that works

[the remark this answers is not in the document]
Comment (sypaq-nexton) [reply Tools/AP_Bootloader/board_types.txt:253]: Thanks! Updated.

[the remark this answers is not in the document]
Comment (WickedShell) [reply Tools/AP_Bootloader/board_types.txt:253]: Fine with us as well.

[the remark this answers is not in the document]
Comment (sypaq-nexton) [reply Tools/AP_Bootloader/board_types.txt:253]: > Fine with us as well.

Thanks! Updated yours as well.

**Review by peterbarker — APPROVED** [review]
LGTM

**Pull request merged**